docs/developer/tutorials/logs/submit-logs-from-http.md

```diff
@@ -0,0 +1,142 @@
+# Submit Logs from HTTP API
+
+## Getting Started
+
+This tutorial assumes you have done the following:
+
+- [set up your environment](../../index.md#getting-started)
+- read the [logs crawler documentation](../../base/logs-crawlers.md)
+- read about the [HTTP capabilities](../../base/http.md) of the base class
+
+Let's say we are building an integration for an API provided by *ACME Inc.*
+Run the following command to create the scaffolding for our integration:
+
+```
+ddev create ACME
```

Comment:
Does the scaffolding for this specific tutorial and http crawler logs need a specific version of ddev or does it work with any version? If so we can try mentioning the ddev version here.

Reply:
Nope, we haven't added any special logs crawler template yet.